README.md

```diff
@@ -79,7 +79,7 @@ Since we're writing a Cloud Native Buildpack, we also need a `buildpack.toml`. U
 file named `buildpack.toml` in the root of your project, right next to `Cargo.toml`.
 
 ```toml
-api = "0.8"
+api = "0.9"
```

Comment:
There are a few more instances of `api = "0.8"` in `libcnb-data/src/buildpack/mod.rs` (rustdocs + tests) that I think we should update to `api = "0.9"` too.

Reply:
Thanks! Fixed via d7e26ee.